package.json

```diff
@@ -8,6 +8,7 @@
     "@rails/webpacker": "5.2.1",
     "bootstrap": "^5.0.0-alpha1",
     "dropzone": "^5.7.2",
+    "popper.js": "^1.16.1",
```

Comment:
while the bootstrap package was already installed, we needed the popper.js package for bootstrap 5.